### PR TITLE
bugherd task #622

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/users-guide/device-management-bundle/monitoring-and-controlling-devices.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ The bottom arrow represents the push connection (from {{< product-c8y-iot >}} to
 * Offline (connection not established) - indicated by a red arrow
 * Not monitored - indicated by a grey arrow
 
-A push connection is an active HTTPS long poll or an MQTT connection from {{< product-c8y-iot >}} to the <kbd>/notification/operations</kbd> API endpoint (not the real time API endpoint).
+A push connection is an active HTTPS long poll connection to the /notification/operations API endpoint or an active MQTT Connection to the MQTT endpoint of Cumulocity IoT.
```
Comment (manasesjesus) [inline content/users-guide/device-management-bundle/monitoring-and-controlling-devices.md:99]:
```suggestion
A push connection is an active HTTPS long poll connection to the <kbd>/notification/operations</kbd> API endpoint or an active MQTT connection to the MQTT endpoint of Cumulocity IoT.
```

File: content/users-guide/device-management-bundle/monitoring-and-controlling-devices.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ The bottom arrow represents the push connection (from {{< product-c8y-iot >}} to
 * Offline (connection not established) - indicated by a red arrow
 * Not monitored - indicated by a grey arrow
 
-A push connection is an active HTTPS long poll or an MQTT connection from {{< product-c8y-iot >}} to the <kbd>/notification/operations</kbd> API endpoint (not the real time API endpoint).
+A push connection is an active HTTPS long poll connection to the <kbd>/notification/operations</kbd> API endpoint or an active MQTT connection to the MQTT endpoint of Cumulocity IoT.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/monitoring-and-controlling-devices.md:99]: Cumulocity IoT > use variable

Comment (rssag) [reply content/users-guide/device-management-bundle/monitoring-and-controlling-devices.md]: done

**Review by manasesjesus — APPROVED** [review]
👍🏼

**Review by MWindel — APPROVED** [review]

**Comment by rssag** [comment]
fixed in develop, 10.15.0, 10.14.0, 10.13.0 and 10.10.0.

**Pull request merged**